subprojects/core-api/src/main/java/org/gradle/api/file/ContentFilterable.java

```diff
@@ -85,9 +87,36 @@
      * Groovy's {@link groovy.text.SimpleTemplateEngine}. This means you can use simple property references, such as
      * <code>$property</code> or <code>${property}</code> in the file. You can also include arbitrary Groovy code in the
      * file, such as <code>${version ?: 'unknown'}</code> or <code>${classpath*.name.join(' ')}</code>
+     * <p>
+     * Note that all escape sequences ({@code \n}, {@code \t}, {@code \\}, etc) are converted to the symbols
+     * they represent, so, for example, {@code \n} becomes newline. If this is undesirable then {@link #expand(Map, Action)}
+     * should be used to disable this behavior:
+     * <pre>
+     *  expand(one: '1', two: 2) {
+     *      escapeBackslash = true
+     *  }
+     * </pre>
```

Comment:
I think this example should be moved into `#expand(Map,Action)` since we don't have an example there and we're telling people to use that method

Reply:
Done